docs/find-options.md

```diff
@@ -367,8 +367,39 @@ will execute following query:
 SELECT * FROM "post" WHERE "currentDate" > NOW()
 ```
 
-> Note: beware with `Raw` operator. It executes pure SQL from supplied expression and should not contain a user input,
- otherwise it will lead to SQL-injection.
+If you want to provide user input, in order to prevent SQL-injection, you can use the native parameters by providing a second argument to the `Raw` function.
```

Comment:
A little more verbose but maybe clearer?  What do you think?

```suggestion
If you need to provide user input, you should not include the user input directly in your query as this may create a SQL injection vulnerability.  Instead, you can use the second argument of the `Raw` function to provide a list of parameters to bind to the query.
```

Reply:
Fixed and commit (doc:) amended